.github/workflows/build-and-test.yml

```diff
@@ -91,6 +91,9 @@ jobs:
           - "5.34"
           - "5.36"
     steps:
+      - name: pin Test::Deep to 1.130 on Perl < 5.12
+        if: ${{ matrix.perl-version == '5.8' || matrix.perl-version == '5.10' }}
+        run: cpanm --notest Test::Deep@1.130
```

Comment:
Just wondering why we use `cpanm` here, but `cpm` in the other places?

Reply:
I think that's historical. I used the one it uses to install from the cpanfile for each of the jobs.

Reply:
We could change that, I suppose. And it might not need the checkout step in Linux either, the other two don't have it.

Reply:
In a later commit, though.